src/json-util.cpp

```diff
@@ -436,6 +436,7 @@ void ju_get_opt_field(const nlohmann::json &j, const K &key, machine_runtime_con
     ju_get_field(j[key], "htif"s, value.htif, path + to_string(key) + "/");
     ju_get_opt_field(j[key], "skip_root_hash_check"s, value.skip_root_hash_check, path + to_string(key) + "/");
     ju_get_opt_field(j[key], "skip_version_check"s, value.skip_version_check, path + to_string(key) + "/");
+    ju_get_opt_field(j[key], "soft_yield"s, value.soft_yield, path + to_string(key) + "/");
```

Comment:
There is a matter of consistency here. Other options are "do this". This option is "this". Do we change the other ones to be "no_root_hash_check" and "no version check", or do we change this to "honor soft yield" or some such?

Reply:
So, nothing to do here now, right? Leave at is?
This PR is not about renaming `skip` flag to `no`, you had your chance on the PR these flags were added :)

Reply:
I guess I'll have to recruit @vfusco. :)